<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>366</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>274</width>
    <height>366</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Anhalteweg </string>
  </property>
  <property name="windowIcon">
   <iconset resource="bremsweg.qrc">
    <normaloff>:/icons/statistics.png</normaloff>:/icons/statistics.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Geschwindigkeit in km/h:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QSpinBox" name="Tempo_spinBox">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="maximum">
         <number>240</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3" colspan="3">
       <widget class="QDial" name="Tempo_dial">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximum">
         <number>240</number>
        </property>
        <property name="notchesVisible">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Mindestabstand (gute Sicht):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" rowspan="2" colspan="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="5">
       <widget class="QLCDNumber" name="lcd_g_mindest">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>23</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="4" rowspan="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Mindestabstand (schlechte Sicht):</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QLCDNumber" name="lcd_s_mindest">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>23</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Reaktionsweg:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3" colspan="2">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="5">
       <widget class="QLCDNumber" name="lcd_rweg">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>23</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>normale Bremsung</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Bremsweg:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>110</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QLCDNumber" name="lcd_n_bweg">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>23</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Anhalteweg:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>110</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <widget class="QLCDNumber" name="lcd_n_aweg">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>23</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Gefahrenbremsung</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Bremsweg:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>110</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QLCDNumber" name="lcd_g_bweg">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>23</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Anhalteweg:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>110</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <widget class="QLCDNumber" name="lcd_g_aweg">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>23</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="bremsweg.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Tempo_spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Tempo_dial</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Tempo_dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Tempo_spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
